lib/moribus/aggregated_behavior.rb

```diff
@@ -48,9 +48,16 @@ def save(*)
 
     # Bang version of #save.
     def save!(*args)
-      save(*args) or raise ActiveRecord::RecordNotSaved
+      save(*args) or raise_record_not_saved_error
     end
 
+    # Raise record not saved
+    def raise_record_not_saved_error
+      args = Rails::VERSION::MINOR < 2 ? [] : ["Failed to save record"]
```

Comment:
This might survive changes to add Rails 5 support, and could conceivably subtly fail in that case. Better to more tightly constrain this predicate:
```ruby
      args =
        (Rails::VERSION::MAJOR == 4 && Rails::VERSION::MINOR < 2) ? [] : ["Failed to save record"]
```
***
Fixing.